### PR TITLE
Feat/quota

### PR DESCRIPTION
**Pull request by Uier**

### CODE REVIEW

File: src/components/courses/ShowHomework.vue [hunk 1]
```diff
@@ -76,18 +76,20 @@
                   <th class="font-weight-bold subtitle-1 text--primary" v-text="'PID'"></th>
                   <th class="font-weight-bold subtitle-1 text--primary" v-text="'Name'"></th>
                   <th class="font-weight-bold subtitle-1 text--primary" v-text="'Type'"></th>
+                  <th class="font-weight-bold subtitle-1 text--primary" v-text="'Quota'"></th>
                   <th class="font-weight-bold subtitle-1 text--primary" v-text="'Score'"></th>
                   <th class="font-weight-bold subtitle-1 text--primary" v-text="'Statistic'"></th>
```
Comment (Bogay) [inline src/components/courses/ShowHomework.vue]: Why not use a `v-for` loop instead?

File: src/pages/courses/problems/Problems.vue [hunk 2]
```diff
@@ -40,6 +40,7 @@
               <th class="font-weight-bold subtitle-1 text--primary">Name</th>
               <th class="font-weight-bold subtitle-1 text--primary">Type</th>
               <th class="font-weight-bold subtitle-1 text--primary">Tags</th>
+              <th class="font-weight-bold subtitle-1 text--primary">Quota</th>
               <th class="font-weight-bold subtitle-1 text--primary">Score</th>
               <th class="font-weight-bold subtitle-1 text--primary">Statistic</th>
```
Comment (Bogay) [inline src/pages/courses/problems/Problems.vue]: And also here can use a `v-for` loop.

File: src/pages/problem/Problem.vue [hunk 3]
```diff
@@ -24,6 +24,11 @@
               </v-btn>
             </v-col>
           </v-row>
+          <v-row class="pl-4" no-gutters>
+            <v-col>
+              <p>Quota: <strong class="text--primary">{{ this.prob.quota==-1 ? 'unlimited' : this.prob.quota-this.prob.submitCount }}</strong></p>
```
Comment (Bogay) [inline src/pages/problem/Problem.vue]: A computed property may be more expressive.

File: src/pages/courses/problems/Problems.vue [hunk 4]
```diff
@@ -62,6 +63,7 @@
                   label small
                 >{{ tag }}</v-chip>
               </td>
+              <td class="subtitle-1" v-text="item.quota==-1 ? 'unlimited' : item.quota-item.submitCount"></td>
```
Comment (Bogay) [inline src/pages/courses/problems/Problems.vue]: A computed property may be more expressive.

File: src/pages/courses/manages/ManageProblems.vue [hunk 5]
```diff
@@ -500,6 +515,7 @@ export default {
           });
         }
       }
+      this.prob.quota = Number(this.prob.quota);
```
Comment (Bogay) [inline src/pages/courses/manages/ManageProblems.vue]: `this.prob.quota = +this.prob.quota` may be more simpler?

**Comment by Uier** [comment]
Sorry, we have to merge it, code beautifying can be post as issue, and i'll fix it

**Pull request merged**